crates/zk-por-core/src/circuit_config.rs

```diff
@@ -39,3 +39,7 @@ pub const HIGH_RATE_CONFIG: CircuitConfig = CircuitConfig {
         num_query_rounds: 12,
     },
 };
+
+pub const RECURSIVE_FACTOR: usize = 64;
```

Comment:
it's in the yml config; can have a struct to includes all this info; better not to put as constant

Reply:
we have to put a const, as this parameter will be used as generic. Hence we cannot dynamically load from config.